lib/build/ProjectBuilder.js

```diff
@@ -30,6 +30,66 @@ class ProjectBuilder {
 	 * 			If the wildcard '*' is provided, only the included tasks will be executed.
 	 */
 
+	/**
+	 * Alternatively to providing plain lists of dependency names to include and exclude, a more complex
+	 * "Dependency Includes" object can be provided to define which dependencies should be part of the build result.
+	 * <br>
+	 * The provided information will be used to compile lists of <code>includedDependencies</code> and
+	 * <code>excludedDependencies</code>, which will be used during the build process.
+	 * <br><br>
+	 * Regular expression-parameters are directly applied to a list of all project dependencies
```

Comment:
Would an example for such a parameter be helpful?

Reply:
Certainly. I'll make a note for that.

But since this API is mostly used internally, I think there is no need to provide that level of documentation right away.